### PR TITLE
SLE-137 Activate C/C++ file suffixes by default in CDT module

### PR DESCRIPTION
**Pull request by dbmeneses**

### CODE REVIEW

**Comment by schoeso** [comment]
Sehr geehrte Damen und Herren,

ich bin derzeit nicht im Büro, werde aber so schnell wie möglich auf Ihre Anfrage reagieren. Am 02.01.2017 bin ich wieder zurück.

Mit freundlichen Grüßen

Dr. Arndt Schönberg

File: org.sonarlint.eclipse.cdt/src/org/sonarlint/eclipse/cdt/internal/CProjectConfigurator.java [hunk 1]
```diff
@@ -25,11 +25,17 @@
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.util.Collection;
+import java.util.Collections;
 import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.List;
 import java.util.Map;
```
Comment (SonarTech) [inline org.sonarlint.eclipse.cdt/src/org/sonarlint/eclipse/cdt/internal/CProjectConfigurator.java]: MINOR Remove this unused import 'java.util.Map'. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)

File: org.sonarlint.eclipse.cdt/src/org/sonarlint/eclipse/cdt/internal/CProjectConfigurator.java [hunk 2]
```diff
@@ -25,11 +25,17 @@
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.util.Collection;
+import java.util.Collections;
 import java.util.HashMap;
```
Comment (SonarTech) [inline org.sonarlint.eclipse.cdt/src/org/sonarlint/eclipse/cdt/internal/CProjectConfigurator.java]: MINOR Remove this unused import 'java.util.HashMap'. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by SonarTech** [comment]
SonarQube analysis reported 3 issues
* MINOR 3 minor

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MINOR [AnalyzeProjectJob.java#L78](https://github.com/SonarSource/sonarlint-eclipse/blob/91d6a1acd1d04fb22948efb64b05b5c5d15f6dbc/org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/internal/jobs/AnalyzeProjectJob.java#L78): Remove this unused import 'org.sonarsource.sonarlint.core.client.api.common.analysis.IssueListener'. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)

**Pull request merged**